README.md

```diff
@@ -0,0 +1,72 @@
+# Selenium + WebdriverIO
+## 1. lekce 
+
+Základní nastavení WebdriverIO s podporu autocomplete do VSCode. Obsahuje jeden "test" v `test/spects/01_uvod.e2e.js`
+
+### Požadavky
+* nainstalovaný git
+* nainstalované [nodejs](https://nodejs.org/en/) 
+  * Můžete zkontrolovat v terminálu příkazem `node -v`
+* webový prohlížeč [Chrome](https://www.google.com/intl/cs_CZ/chrome/)
+* [Visual Studio Code](https://code.visualstudio.com/)
+
+### Instalace 
+1. Vytvoř si složku, kde chceš mít všechny materiály - doporučuji něco ve stylu `Dokumenty/czechitas/automatizace`
+2. V terminálu jdi do vytvořené složky - pomocí příkazu `cd <dana slozka>`
+2. Stáhni si repozitář do svého počítače pomocí `git clone git@github.com:czechitas-da2020/01_automation.git`
+3. V daném adresáři pusť `npm install` 
+    * Tenhle příkaz Ti nainstaluje všechny potřebné balíčky, které pak budeme používat. 
+3. Otevři si VSCode a otevři v něm repozitář - `Dokumenty/czechitas/automatizace/01_automation`
```

Comment:
Monza bych napsal a otevrni v nem **slozku** at je jasne, ze na PC je to vedene jako slozky.